ext/session/session.stub.php

```diff
@@ -43,35 +43,45 @@ function session_commit(): bool {}
  * @param callable|object $open
  * @param callable|bool $close
  */
-function session_set_save_handler($open, $close = UNKNOWN, callable $read = UNKNOWN, callable $write = UNKNOWN, callable $destroy = UNKNOWN, callable $gc = UNKNOWN, callable $create_sid = UNKNOWN, callable $validate_sid = UNKNOWN, callable $update_timestamp = UNKNOWN): bool {}
+function session_set_save_handler(
+    $open,
+    $close = UNKNOWN,
+    callable $read = UNKNOWN,
+    callable $write = UNKNOWN,
+    callable $destroy = UNKNOWN,
+    callable $gc = UNKNOWN,
+    callable $create_session_id = UNKNOWN,
+    callable $validate_session_id = UNKNOWN,
+    callable $update_timestamp = UNKNOWN
+): bool {}
 
 function session_cache_limiter(?string $cache_limiter = null): string|false {}
 
 function session_cache_expire(?int $new_cache_expire = null): int|false {}
 
-function session_set_cookie_params(array|int $lifetime_or_options, ?string $path = null, ?string $domain  = null, ?bool $secure = null, ?bool $httponly = null): bool {}
+function session_set_cookie_params(array|int $lifetime_or_options, ?string $path = null, ?string $domain = null, ?bool $secure = null, ?bool $httponly = null): bool {}
 
 function session_start(array $options = []): bool {}
 
 interface SessionHandlerInterface
 {
     /** @return bool */
-    public function open(string $save_path, string $session_name);
+    public function open(string $path, string $name);
 
     /** @return bool */
     public function close();
 
     /** @return string */
-    public function read(string $key);
+    public function read(string $id);
 
     /** @return bool */
-    public function write(string $key, string $val);
+    public function write(string $id, string $value);
 
     /** @return bool */
-    public function destroy(string $key);
+    public function destroy(string $id);
 
     /** @return int|bool */
-    public function gc(int $maxlifetime);
+    public function gc(int $lifetime);
```

Comment:
Does it still have the same meaning? Could we use TTL? I think lifetime has too widespread usage to change it.

Reply:
I like the TTL idea, but there's also the consideration that this is the value specified by `session.gc_maxlifetime`.

Reply:
I left this as `$lifetime`, but I'm ok to revert this to `$maxlifetime` or `$max_lifetime` if you prefer these.

Reply:
Mild preference for `$max_lifetime` from my side.

Reply:
OK, seems good for me as well. And I've just pushed a commit with this.